libmachine/drivers/plugin/register_driver.go

```diff
@@ -20,6 +21,7 @@ var (
 
 func RegisterDriver(d drivers.Driver) {
 	if os.Getenv(localbinary.PluginEnvKey) != localbinary.PluginEnvVal {
+		fmt.Fprintln(os.Stdout, fmt.Sprintf("API version %d", version.ApiVersion))
```

Comment:
Seems reasonable, but why not just change the below line to a `Fprintf` instead and including it in the multi-line message?  I'd prefer this to go to STDERR as well, since invoking a plugin binary directly is intended to be an error.